packages/field-highlighter/test/field-highlighter.test.js

```diff
@@ -157,28 +152,22 @@ describe('field highlighter', () => {
         addUser(user2);
         removeUser(user2);
         removeUser(user1);
-        expect(highlighter.user).to.equal(null);
+        expect(highlighter.user).to.be.null;
       });
 
       it('should reset user when multiple users are removed', () => {
         setUsers([user1, user2]);
         setUsers([]);
-        expect(highlighter.user).to.equal(null);
+        expect(highlighter.user).to.be.null;
       });
     });
 
     describe('overlay', () => {
-      afterEach(() => {
```

Comment:
**note:** The `user-tags` component already closes its child overlay once the component is disconnected from the DOM.